soh/soh/Enhancements/audio/AudioEditor.cpp

```diff
@@ -308,7 +308,10 @@ void DrawTypeChip(SeqType type) {
 
 void DrawSfxEditor(bool& open) {
     if (!open) {
-        CVarSetInteger("gAudioEditor.WindowOpen", 0);
+        if (CVarGetInteger("gAudioEditor.WindowOpen", 0)) {
```

Comment:
Do we need these cvar checks? If we used to be setting it to 0 here can't we just clear it?

Reply:
Ah. I see what arches said now. We still need the check to decide if we want to fire off the request to save on next tick. I feel like there should be a cleaner way to handle this but this works

Reply:
Yeah, this is a necessary evil to alleviate the fact that these blocks in every window are running every frame. I think the case could be made that the `DrawFunc` of a window shouldn't be called at all when a window isn't open, but there's probably some sort of ImGui or LUS reasoning behind it.

Reply:
Regardless, this assumes the cvar read +  condition check every frame is cheaper than a cvar write every frame, I'd wager this is correct?